tests/qnn/test_qnn_torch.py

```diff
@@ -74,13 +111,18 @@ def test_no_torch(self, get_circuit, monkeypatch):
                 TorchLayer(c, w)
 
     @pytest.mark.parametrize("n_qubits, output_dim", indices_up_to(1))
-    def test_no_input(self, get_circuit):
+    def test_no_input(self):
         """Test if a TypeError is raised when instantiated with a QNode that does not have an
         argument with name equal to the input_arg class attribute of TorchLayer"""
-        c, w = get_circuit
-        del c.func.sig[qml.qnn.torch.TorchLayer._input_arg]
+        dev = qml.device("default.qubit", wires=1)
+        weight_shapes = {"w1": (3, 3), "w2": 1}
+
+        @qml.qnode(dev, interface="torch")
+        def circuit(w1, w2):
+            return qml.expval(qml.PauliZ(0))
```

Comment:
Makes more sense to put a user facing example here, rather than monkeypatching an existing functions inspected signature.